webapp/packages/plugin-authentication-administration/src/Administration/Users/UserForm/Info/UserFormInfoPart.ts

```diff
@@ -85,6 +87,10 @@ export class UserFormInfoPart extends FormPart<IUserFormInfoState, IUserFormStat
       this.initialState.userId = user.userId;
       this.formState.setMode(FormMode.Edit);
     }
+  }
+
+  protected override async saveChanges(): Promise<void> {
+    await this.createUser();
```

Comment:
If you want to create a separate function, you can remove 
```
this.initialState.userId = user.userId;
 this.formState.setMode(FormMode.Edit);
```

and move this.formState.mode === FormMode.Create check to saveChanges method, so you will always create user when you run createUser method

```
if(this.formState.mode === FormMode.Create) {
   const user = await this.createUser();
   this.initialState.userId = user.userId;
   this.formState.setMode(FormMode.Edit);
}
```